[Do not merge] Remove beta notices for MLM GA (#804)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/change-requests/manage.mdx

Open a workspace and select **Change requests** from the side navigation to display a list of that workspace's active change requests. You can view the full description of a change request by clicking on that request. If someone creates a change request on multiple workspaces, each workspace has its own independent copy of that change request.

> If unarchived change requests exist on a workspace, the change request navigation will have a badge indicating how many.

Reviewer: this bit

Author: Small rephrase for flow, but otherwise great!
```suggestion
The change request navigation displays a badge indicating how many unarchived change requests exist on a workspace.
```
